src/features/OnHover.js

```diff
@@ -18,6 +18,10 @@ var OnHover = Feature.extend('OnHover', {
      */
     handleMouseMove: function(grid, event) {
         var hoverCell = grid.hoverCell;
+        // VC-5715 this is added for quickly repaint the images incase the button is hovered
+        if (event.mousePointInLeftClickRect || event.mousePointInRightClickRect) {
+            grid.repaint()
```

Comment:
When there is nothing change on the data shape/data update, the "getCellConfig" will only be called once a second.
under the method "startDataModelCheckLoop"

This is do the repaint incase the mouse is inside the buttons